src/pybind/mgr/dashboard/frontend/e2e/cluster/osds.e2e-spec.ts

```diff
@@ -32,4 +33,51 @@ describe('OSDs page', () => {
       expect(osds.getTabText(1)).toEqual('Overall Performance');
     });
   });
+
+  describe('check existence of fields on OSD page', () => {
+    it('should check that number of rows and count in footer match', () => {
+      osds.navigateTo();
+      osds
+        .getTableCount()
+        .getText()
+        .then((tableCount) => {
+          tableCount = tableCount.slice(13, -6); // Grabs number of hosts from table footer
+          expect(osds.getTableRows().count()).toMatch(tableCount);
+        });
+    });
+    it('should verify that selected footer increases when an entry is clicked', () => {
+      osds.navigateTo();
+      $$('.datatable-body-cell-label')
```

Comment:
Please declare the access to elements in page object file [[1](http://www.protractortest.org/#/style-guide#use-page-objects-to-interact-with-page-under-test)].
Generic table elements should go into the `helper.po.ts`.
OSD specific elements should go to `osds.po.ts`

Reply:
All changes should be addressed